enterprise/coderd/groups.go

```diff
@@ -408,7 +408,7 @@ func (api *API) groupsByOrganization(rw http.ResponseWriter, r *http.Request) {
 // @Tags Enterprise
 // @Param organization query string true "Organization ID or name"
 // @Param has_member query string true "User ID or name"
-// @Success 200 {array} codersdk.Group
+// @Success 200 {array} codersdk.GroupWithOrganizationInfo
```

Comment:
also fix this on the api above, `groupsByOrganization`

Reply:
good catch!